app/views/external_users/claims/case_details/_case_type_fields.html.haml

```diff
@@ -6,8 +6,7 @@
         = t('.stage_type_hint')
       = validation_error_message(@error_presenter, :case_type)
 
-    = f.select :case_type_id, @case_type_stages.map{ |ct| [ct.name, ct.id, { data: { 'is-fixed-fee': ct.is_fixed_fee?, 'requires-cracked-dates': ct&.requires_cracked_dates?, 'requires-retrial-dates': ct&.requires_retrial_dates?, 'requires-trial-dates': ct&.requires_trial_dates? } }] }, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete', id: 'case_type', 'aria-label': t('.stage_type') }
-
+    = f.select :case_stage_id, @case_stages.map{ |cs| [cs.description, cs.id, { data: { 'is-fixed-fee': cs.is_fixed_fee?, 'requires-cracked-dates': cs&.requires_cracked_dates?, 'requires-retrial-dates': cs&.requires_retrial_dates?, 'requires-trial-dates': cs&.requires_trial_dates? } }] }, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete', id: 'case_type', 'aria-label': t('.stage_type') }
```

Comment:
This seems like a good candidate to pull out to a presenter, but MVP! :)

Reply:
yes, indeed, peter already highlighted the length of the input text